message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -848,6 +848,30 @@
         <description>SPI Device operation</description>
       </entry>
     </enum>
+    <enum name="DEEPSTALL_STAGE">
+      <description>Deepstall flight stage</description>
+      <entry name="DEEPSTALL_STAGE_FLY_TO_LANDING" value="0">
```

Comment:
there was recently a huge reformatting to the xml which moved the value to the front. Mind swapping them so it matches the rest of the file?